eth/backend.go

```diff
@@ -579,6 +600,15 @@ func (s *Ethereum) Protocols() []p2p.Protocol {
 	if s.lesServer != nil {
 		protos = append(protos, s.lesServer.Protocols()...)
 	}
+
+	// /Quorum
+	// add additional quorum consensus protocol if set and if not set to "eth", e.g. istanbul
+	if quorumConsensusProtocolName != "" && quorumConsensusProtocolName != protocolName {
```

Comment:
Q: in which situation `quorumConsensusProtocolName` could get the value `eth`?

Reply:
In the case where the consensus engine return `eth` as the consensus protocol, e.g. `clique`.